src/redux/types/typesStore.ts

```diff
@@ -0,0 +1,43 @@
+export type TToken = null | string;
+export type TRefreshToken = null | string;
+export type TSid= null|string
+export type TIsLoggedIn = boolean
+export type TIsFetchingCurrentUser = boolean;
+export type TId = null | string;
+export type TTitle = undefined | string;
+
+
+export interface IUser {
+    id: null | string,
+    email: null|string
+}
+
+export interface IAuthInitialState{
+    user: IUser,
+    token: TToken,
+    refreshToken: TRefreshToken,
+    sid: TSid,
+    isLoggedIn: TIsLoggedIn,
+    isFetchingCurrentUser: TIsFetchingCurrentUser
+}
+
+export interface IRefreshToken{
+    newAccessToken: TToken;
+    newRefreshToken:TRefreshToken;
+    newSid: TSid;
+}
+
+export interface IProjectState{
+    _id?: string;
```

Comment:
Why did you use underscore before id?

Reply:
That was a backend problem. It gets "_id" and "id" in different requests